content/800-data-platform/050-data-proxy/900-considerations-limitations.mdx

```diff
@@ -0,0 +1,81 @@
+
+We are looking to expand the number of providers and regions in the future. If a region or provider is not yet supported, you can [submit a request](https://pris.ly/proxy-new-region). This will help us prioritize providers and regions to add next.
+
+### Recommended concurrent connections
+
+We recommend that you use a database that supports at least 20 concurrent connections.
+Setting a concurrency limit on AWS Lambda should not be required, but you should be aware that, if necessary, requests will be queued until they can be processed.
+
+For more information about configuring a maximum number of connections to the database, see [Manage the number of database connections](/data-platform/data-proxy/use-data-proxy#manage-the-number-of-database-connections).
+
+### Idle connections to the database
+
+Data Proxy keeps several connection pools running to maintain open database connections even when no queries run from your application. The connection pools and any remaining idle connections close after six minutes of inactivity (or when no incoming queries are sent the database).
+
+The number of idle connections to the database depends on the number of running connection pools at any point in time. 
+In each of the supported locations, the Data Proxy architecture includes multiple clusters that maintain idle connections to the database.
+
+## Limitations
+
+### Prisma Migrate and Introspection do not work with a Data Proxy connection string
+
+You cannot use specific Prisma CLI commands, such as `prisma migrate` and `prisma db pull` with a Data Proxy connection string.
+
+You can work around this issue when you [specify a direct database connection string for each Prisma CLI command](/data-platform/data-proxy/prisma-cli-with-data-proxy) that you need.
+
+### Data Proxy concurrency limits
+
+Depending on your [project plan](/data-platform/billing/plans-and-quotas), the Prisma Data Proxy provides a different [limit of concurrent proxy instances](/data-platform/about/limits-and-status#concurrency-limits-of-the-data-proxy).
+
+The performance of a single proxy instance depends on a large number of variables:
+
+- the type of queries
+- latencies and deployment locations
+- the performance of your database
+- ...
+
+If you are seeing `429` HTTP errors in your logs, this probably means that you are hitting the concurrency limits.
+
+[Contact Support](/data-platform/contact-support) if you are hitting the concurrency limits and have trouble scaling your Prisma application.
+
+### Unsupported Prisma Client features
+
+#### Metrics and Tracing preview features not supported
+Data Proxy does not yet support the following Prisma Client preview features:
+
+- [Metrics](/concepts/components/prisma-client/metrics)
+- [OpenTelemetry tracing](/concepts/components/prisma-client/opentelemetry-tracing)
+
+To use Data Proxy in such cases, you must first remove `metrics` and `tracing` from the `previewFeatures` section of your `schema.prisma` file.
```

Comment:
```suggestion
To use Data Proxy, you must first remove `metrics` and `tracing` from the `previewFeatures` section of your `schema.prisma` file if they are present.
```

I think this is clearer - I understood the sentence as written to be providing a workaround - "if you want to use Data Proxy with these features".